### PR TITLE
done

### PR DESCRIPTION
**Pull request by savchuksavchuk**
[DEMO LINK](https://savchuksavchuk.github.io/layout_snake/)
[TEST REPORT LINK](https://savchuksavchuk.github.io/layout_snake/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
+body {
+  --snake-height: 300px;
+  margin: 0;
+}
+
+.snake {
+  max-width: 1199px;
```
Comment (alexandra-protyanova) [inline src/style.css:7]: your snake should stretch for all width of the screen

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+}
+
+.snake-item {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+  min-width: var(--snake-height);
+  height: var(--snake-height);
+  font-family: Arial, Helvetica, sans-serif;
```
Comment (alexandra-protyanova) [inline src/style.css:20]: move to tag selector `html`

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  color: #fff;
+  font-size: 100px;
+}
+
+.snake-item-1 {
+  background-color: rgb(100%, 0%, 0%);
+  order: 1;
```
Comment (alexandra-protyanova) [inline src/style.css:27]: it's already has first order

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+  .snake-item-4 {
+    order: 3;
+  }
+  .snake-item-5 {
+    order: 5;
+  }
+  .snake-item-6 {
+    order: 6;
+  }
+}
```
Comment (alexandra-protyanova) [inline src/style.css:83]: delete duplicating

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+  color: #fff;
+  font-size: 100px;
+}
+
+.snake-item-1 {
+  background-color: rgb(100%, 0%, 0%);
+  order: 1;
+}
+
+.snake-item-2 {
```
Comment (alexandra-protyanova) [inline src/style.css:30]: same

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+  background-color: rgb(80%, 0%, 0%);
+  order: 2;
+}
+
+.snake-item-3 {
+  background-color: rgb(60%, 0%, 0%);
+  order: 3;
```
Comment (alexandra-protyanova) [inline src/style.css:37]: same

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 2 above)
Comment (savchuksavchuk) [reply src/style.css]: i can't move it to html selector because style property is banned

**Pull request closed without merging**